members/templates/members/edit.html

```diff
@@ -2,6 +2,10 @@
 {% block title %}Edit Member{% endblock %}
 {% block content %}
     <p style="color:red">{{ error }}</p>
+    {% if warning %}
+        <p style="color:red">The user <a href="{% url 'members.view' warning.id 0 %}">{{ warning.name }}</a> has a similar Student #.
```

Comment:
Very pedantic, but maybe use 'member' instead of 'user', since we're talking about registring members here :).